### ISSUE TITLE
train.csv and y_train.csv, what these files are used for and what they contain?  Thanky!

### ISSUE THREAD

**Issue by junzai0215** [crazyinter/Gene2Vec#1]
There is "train_500bp.csv" in readme. But I have not found it in Seq2Vec Project. Besides this, I have not found these files referred in train.py:
X_train=np.loadtxt(open("train.csv","rb"),delimiter=",",skiprows=0)
y_train=np.loadtxt(open("y_train.csv","rb"),delimiter=",",skiprows=0)

train.csv and y_train.csv, what these files are used for and what they contain?  Thanky!

**Comment by jamesriddellv** [comment]
I'm not a developer, but I think the train.csv and y_train.csv are for if you are training the model on your own database within the Train.py. train.csv would be your feature matrix X, and y would be a vector of the values you are trying to predict. All items in a row of X correspond to a row of y. If you look at the `Seq2Vec_train&test.py ` script, you just need to rename `train_data` in `g=open('train_data','r')` to the training set of your choice, and `f=open('test_data','r')` as your test set. So you could set the training set to train_500bp.csv, and test_data as the contigs you're trying to get a score for. 

Please correct me if I'm wrong! But I will try this after getting my into the same format as the example real test set.
